src/cronjobs.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import {CronJob} from 'cron'
+import Offer from './models/Offer'
+import forms from './models/forms'
+import UserAction, {
+  ACTION_AUTOMATICALLY_DELETED_EXPIRED,
+  ACTION_AUTOMATICALLY_DELETED_NOT_CONFIRMED
+} from './models/UserAction'
+
+const TIMEZONE = 'Europe/Berlin'
```

Comment:
This should be changed to the system timezone. moment.js should be able to handle this.